src/client/tweetAnalysis.spec.js

```diff
@@ -30,6 +30,37 @@ describe('#TweetAnalysis', () => {
       mostFrequentWords(tweets).should.deep.equal(exampleFrequentWords);
     });
 

```

Comment:
I'd say we should, now that they're all unique, each one will only appear once anyway. Do you think we should remove them as they're not "words"?